r/R/dplyr-count.R

```diff
@@ -25,12 +25,13 @@ count.arrow_dplyr_query <- function(x, ..., wt = NULL, sort = FALSE, name = NULL
   }
   out <- dplyr::tally(out, wt = {{ wt }}, sort = sort, name = name)
 
-  # Restore original group vars
   gv <- dplyr::group_vars(x)
-  if (length(gv)) {
+  if (rlang::is_empty(gv)) {
```

Comment:
Thanks for this! For rlang functions, we prefer to import them so we don't have to use `rlang::`. Could you please add `is_empty` in to the imported rlang functions in [`arrow-package.R`](https://github.com/apache/arrow/blob/ade1027c3f6711374c6ccccff5ce068e0476664d/r/R/arrow-package.R#L22-L26), remove the `rlang::` here, and run `devtools::document()`? Thanks!

Reply:
That makes sense. Thanks for the feedback!